modules/rgbd/src/hash_tsdf.hpp

```diff
@@ -33,8 +33,10 @@ class HashTSDFVolume : public Volume
     int volumeUnitResolution;
     float volumeUnitSize;
     bool zFirstMemOrder;
+    Point3i volumeDims;
+    Vec4i volDims;
```

Comment:
Please rename one of them (or both?) to be more self-descriptive
For example, `volDims` to `volStrides`

Reply:
done